docs/UserGuide.adoc

```diff
@@ -896,7 +897,10 @@ Edits the session at index 1 to have:
 
 You can use this command to mark a session as done.
 
-The session will automatically appear as a new claim in the *Claims Vieww*.
+If the session marked done has a recurring period, a new session with the updated date
+and timing will be added to Session View.
+The session marked done will be removed from *Session View* and
```

Comment:
Maybe the two could be swapped? The session marked done moves to claims, then if recurring repeat.